Resources/Private/Javascript/Components/Orders.js

```diff
@@ -21,48 +22,52 @@ const Orders = () => {
   },[])
 
   return (
-    <div>
+    <div className='c-orders'>
       <Overlay loading={loading} />
 
       {orderItems.length > 0 ?
-        <ul>
+        <ul className='c-orders__list'>
           {orderItems.map(order => (
-            <li key={order.id}>
-              <dl>
-                <dt>Order:</dt>
-                <dd>{order.orderNumber}</dd>
+            <li className='c-orders__order' key={order.id}>
+              <dl className='c-orders__order-details'>
```

Comment:
There is a lot of similar markup to e.g. the cart summary, can we find a component, which is easily usable? Like `<DefinitionList><DefinitionList.DefinitionItem label={Order} value={order.orderNumber} /><DefinitionList>`